src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/ccd/CcdClient.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.ccd;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+@Service
+public class CcdClient {
+
+    private static final Logger log = LoggerFactory.getLogger(CcdClient.class);
+
+    private static final String AWAITING_DCN_PROCESSING_FIELD_NAME = "awaitingPaymentDCNProcessing";
+    private static final String COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID = "completeAwaitingPaymentDCNProcessing";
+
+    private final CoreCaseDataApi ccdApi;
+    private final CcdAuthenticatorFactory authenticatorFactory;
+
+    public CcdClient(
+        CoreCaseDataApi ccdApi,
+        CcdAuthenticatorFactory authenticatorFactory
+    ) {
+        this.ccdApi = ccdApi;
+        this.authenticatorFactory = authenticatorFactory;
+    }
+
+    public void completeAwaitingDcnProcessing(
+        String exceptionRecordCcdId,
+        String service,
+        String jurisdiction
+    ) {
+        log.info(
+            "Completing awaiting payment DCN processing. Exception record ID: {}, service: {}",
+            exceptionRecordCcdId,
+            service
+        );
+
+        CcdAuthenticator authenticator = authenticatorFactory.createForJurisdiction(jurisdiction);
+        String caseTypeId = service.toUpperCase() + "_ExceptionRecord";
+        String eventId = COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID;
+
+        StartEventResponse startEventResponse =
+            startEvent(authenticator, jurisdiction, caseTypeId, exceptionRecordCcdId, eventId);
+
+        Event event = Event
+            .builder()
+            .summary("Complete payment DCN processing")
+            .id(startEventResponse.getEventId())
+            .build();
+
+        CaseDataContent caseDataContent = CaseDataContent.builder()
+            .data(ImmutableMap.of(AWAITING_DCN_PROCESSING_FIELD_NAME, "No"))
```

Comment:
is there a constant for `No` in this project?

Reply:
There's no such constant in this project. This is the only occurrence.